lms/djangoapps/badges/models.py

```diff
@@ -256,7 +266,9 @@ class CourseEventBadgesConfiguration(ConfigurationModel):
     )
 
     def __unicode__(self):
-        return u"<CourseEventBadgesConfiguration ({})>".format(u"Enabled" if self.enabled else u"Disabled")
+        return HTML(u"<CourseEventBadgesConfiguration ({})>").format(
+            HTML(u"Enabled") if self.enabled else HTML(u"Disabled")
+        )
```

Comment:
Is this HTML or XML?

Reply:
Neither, it's just text that should only be used in debugging that we don't want escaped. We want it to read `<CourseEventBadgesConfiguration (Enabled)>` not `&lt;CourseEventBadgesConfiguration (Enabled)&gt;`. I should probably change "Enabled" and "Disabled" to use `TEXT()` instead of `HTML()`, though.